<?xml version="1.0"?>
<launch>
<rosparam command="load" file="$(find ur5_intpro)/config/ur5_params.yaml" />

<arg name="ur5_ip" default="192.168.1.103" />
<arg name="ur5_port" default="30003" />
<arg name="r2fg_port" default="/dev/ttyUSB0"/>

<param name="ur5/ip" value="$(arg ur5_ip)" />
<param name="ur5/port" value="$(arg ur5_port)" />
<param name="r2fg/port" value="$(arg r2fg_port)" />

<node name="ur5" type="ur5" pkg="irl_robots" output="screen"/>
<node name="r2fg" type="r2fg" pkg="irl_robots" output="screen"/>

<!-- <arg name="server" default="239.255.42.99"/> -->
<arg name="server" default="192.168.1.116"/>
<node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
    server: $(arg server)
    port: 3883
    update_frequency: 120.0
    frame_id: world
    # Use the VRPN server's time, or the client's ROS time.
    use_server_time: false
    broadcast_tf: true
    # Must either specify refresh frequency > 0.0, or a list of trackers to create
    refresh_tracker_frequency: 1.0
    #trackers:
    #- FirstTracker
    #- SecondTracker
    </rosparam>
</node>
<!-- <node pkg="tf" type="static_transform_publisher" name="opti_static_broadcaster" args="0 0 0 0 0 0 base world 20" /> -->
<node name="intprim_framework_ros" pkg="ur5_intpro" type="optitrack_tf.py" output="screen" respawn="true" />

<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
      <arg name="address" value="10.218.101.9"/>
</include>


</launch>
